tests/file_storage/tests.py

```diff
@@ -417,7 +461,8 @@ def get_available_name(self, name, max_length=None):
         return name
 
 
-class CustomStorageTests(FileStorageTests):
```

Comment:
aren't we losing a lot of tests if `FileStorageTestsBase` only includes setUp/tearDown now?